reference/configuration/doctrine.rst

```diff
@@ -269,9 +269,14 @@ you can control. The following configuration options exist for a mapping:
 ........
 
 One of ``annotation`` (for PHP annotations; it's the default value),
-``attribute`` (for PHP attributes), ``xml``, ``yml``, ``php`` or
+``attribute`` (for PHP attributes), ``xml``, ``php`` or
 ``staticphp``. This specifies which type of metadata type your mapping uses.
 
+.. deprecated:: ORM 3.0
```

Comment:
@cosminsandu I think you should be able to add an entry to the list in `.doctor-rst.yaml`